cmd/callSC/main.go

```diff
@@ -1,36 +1,24 @@
 package main
```

Comment:
This is a test @gregLibert , can you see me ?

Reply:
yes